### DISCUSSION TITLE
How to set up 301 redirect?

### DISCUSSION THREAD

**Discussion by sigridchen** [awdr74100/awdr74100.github.io#99] [Q&A] [answered]
I also use github pages to deploy my blog, but currently I plan to migrate it to vercel. Can you provide a tutorial article on setting up a 301 redirect?

**Comment by awdr74100** [comment, marked as answer]
No. It is not possible.

Your site on GitHub Pages is static, so you don't have any control over the server.

See [stackoverflow](https://stackoverflow.com/questions/5302663/permanent-redirect-from-github-gh-pages).

**Reply by sigridchen** [reply]
Got it, thank you.
